dasharo-security/tpm2-commands.robot

```diff
@@ -225,6 +225,39 @@ TPMCMD011.001 Performing HMAC operation on the file (Ubuntu 22.04)
     Should Contain    ${out1}    hmac.out
     Should Not Contain    ${out2}    hmac.out
 
+TPMCMD012.001 Encrypt and Decrypt non-rootfs partition (Ubuntu 22.04)
+    [Documentation]    Test encrypting and decrypting non-rootfs partition using
+    ...    TPM.
+    # Check:
+    Check If Package Is Installed    cryptsetup
```

Comment:
@DaniilKl This keyword returns TRUE or FALSE depending on if package is installed. You should fail if returned value is FALSE.

Reply:
@m-iwanicki This should be checked in Ansible. Waiting for Ansible support to be released.